lib/usedby/gems.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
```

Comment:
Everything works well with the change. I might prefer to have the filename be `GEMFILE` or `DUMMY_GEMFILE` instead of `gems.rb` to make it more explicit, but this way works fine as well.

Reply:
Yeah right now `Gemfile` is the norm but actually in the future it will be `gems.rb`.
https://depfu.com/blog/2017/09/06/gemfiles-new-clothes